lib/util/git/index.ts

```diff
@@ -543,16 +544,22 @@ export function getBranchList(): string[] {
 }
 
 export async function isBranchBehindBase(branchName: string): Promise<boolean> {
-  await syncGit();
+  const { currentBranchSha } = config;
+
+  let isBehind = getCachedBehindBaseResult(branchName, currentBranchSha);
+  if (isBehind !== null) {
+    return isBehind;
+  }
   try {
+    await syncGit();
```

Comment:
`syncGit` has its own error handling, so what's the benefit of moving to another `try-catch` block? 🤔

Reply:
Mistake actually.. I didn't know hoe the try-catch flow worked. Thanks for pointing out.♥